overlord/snapstate/snapstate.go

```diff
@@ -843,6 +845,7 @@ func applyAutoAliasesDelta(st *state.State, delta map[string][]string, op string
 
 		snapsup := &SnapSetup{
 			SideInfo: &snap.SideInfo{RealName: snapName},
+			// TODO parallel-install: include instance key
```

Comment:
we probably want a more general TODO about the whole alias code vs instate keys and parallel installs

Reply:
Tweaked the comment a little bit.